docker_mirror.py

```diff
@@ -108,7 +113,7 @@ def set_docker_config_ce(mirror):
     config_dict={}
     if os.path.exists(docker_config) != True:
         # if file not exist, create it first.
-        os.mknod(docker_config, 0644)
+        os.mknod(docker_config, 0o644)
```

Comment:
What does the 'o' mean?

Reply:
For Python 2.6, the changed and new token definitions will be:

integer        ::=     decimalinteger | octinteger | hexinteger |
                       bininteger | oldoctinteger

**octinteger     ::=     "0" ("o" | "O") octdigit+**